docs/source/documentation/index.html.md.erb

```diff
@@ -1,5 +1,19 @@
-<!-- Do not delete this file it is the website entry point -->
+<!-- Masthead and call to action -->
+<main id="main" role="main">
+    <h1>Manage Your Civil Cases</h1>
+</main>
 
-# Manage your Civil Cases (MCC)
+## Getting Started
+- [Getting Started](documentation/getting-started.html)
 
-**Start here**
+## Features
+- [Features](documentation/features.html)
+
+## Testing
+- [Testing](documentation/testing.html)
+
+## Deployment
+- [Deployment](documentation/deployment.html)
```

Comment:
We can also more these readme's into the this content's page:

- `locales/README.md`
- `src/scripts/helpers/README.md`